src/app/banner/IssueBanner.tsx

```diff
@@ -20,41 +26,156 @@ export default function IssueBanner() {
     setIsMinimized(true);
   };
 
+  React.useEffect(() => {
+    const getGameTwoGiveData = () => {
+      fetch(
+        `https://www.helpmakemiracles.org/api/1.2/participants/376997?_=${Date.now().toString()}`
```

Comment:
You'll need to add `www.helpmakemiracles.org` to our CSP policy in `content-security-policy.js`.